### ISSUE TITLE
Are we actually muting people who are uncalibrated?

### ISSUE THREAD

**Issue by gwillen** [dspeyer/ritualEngine#145]
To skip calibration in ritual engine, you have to click a button that says "don't let anybody hear me", but it doesn't seem like we actually do anything to prevent audio from going out anyway.

(Also, ritual engine doesn't provide users with a mute button in case something noisy happens when they're trying to sing; that's kind of separate, but it probably should.)

**Comment by dspeyer** [comment]
This button ensures they will be in the last bucket, though they still
might be heard via the /bb interface or in the saved recording if anyone
ever listens to that.  There was also a rumor that BBS itself would refuse
to send audio if calibration hadn't been done, but if you haven't heard of
that it probably isn't true.

**Comment by gwillen** [comment]
As far as I can tell, BBS will not do that; demo.js (the BBS frontend) manages that.

In https://github.com/dspeyer/ritualEngine/pull/153 which saves and loads calibration, I also added

`mycontext.send_ignore_input(true);`

if calibration fails, which will tell BBS to put us in listen-only mode (this can ONLY be done before SingerClient is created, and will not just mute the mic, but put it into a mode where it sends no audio to the server at all, and I believe will never do so unless the BucketContext is closed and recreated, or it's reversed by calling `send_ignore_input(false).`

Also, I believe the prior code only counted calibration as 'failed' if they clicked the button, and not if the calibrator failed to hear anything. I changed the code to listen to the calibrator's failure indicator. It's possible that this is too stringent for our needs and we want something in between.
